src/raw/cgroup.go

```diff
@@ -206,6 +206,7 @@ const nanoSecondsPerSecond = 1e9
 // statistics line and then sums up the first seven fields
 // provided. See `man 5 proc` for details on specific field
 // information.
+// TODO: we should inject the route of /proc/stat in order to be able to mock the file and test the method
```

Comment:
As discussed, maybe we can take the chance to create a new issue to address the refactor to make this _testable_.

Reply:
Yes, I will add this as an issue and I will add another issue with my first findings on needed refactors. For sure we will find many other on the process.

Reply:
done